code/drasil-code/Language/Drasil/CodeSpec.hs

```diff
@@ -111,7 +111,7 @@ codeSpec SI {_sys = sys
 
 -- Converts a chunk with a defining relation to a QDefinition
 relToQD :: ExprRelat c => ChunkDB -> c -> QDefinition
```

Comment:
Note that a big "step 2" of this change should be to remove `ExprRelat` entirely.  `relToQD` as a piece of functionality will still exist, but it will be more of a printing combinator instead.